volmdlr/faces.py

```diff
@@ -4810,6 +4810,7 @@ def get_closed_contour_divided_faces_inner_contours(list_faces, new_contour):
                 if not new_face.surface2d.inner_contours:
```

Comment:
In this method, you don't think that there is an indentation problem ?
![image](https://user-images.githubusercontent.com/68638483/199454556-48fafc56-f0f8-4b92-bebd-8612d04f3686.png)